bundles/org.openhab.binding.dirigera/README.md

```diff
@@ -0,0 +1,781 @@
+Standard duration is seconds if raw number is sent as command. 
+See [Motion Sensor Rules](#motion-sensor-rules) for further examples.
+  
+Mappings for `schedule`
+
+- 0 : Always, sensor is always active
+- 1 : Follow sun, sensor gets active at sunset and deactivates at sunrise 
+- 2 : Schedule, custom schedule with manual start and end time
+
+If option 1, follow sun is selected ensure you gave the permission in the IKEA smart home app to use your GPS position to calculate times for sunrise and sunset.
+
+See [Light Controller](#light-controller) for light-preset`.
+
+## Motion Light Sensor
+
+Sensor detecting motion events and measures light level.
+Same channels as [Motion Sensor](#motion-sensor) with an additional `illuminance` channel.
+
+| Channel               | Type                  | Read/Write | Description                                  | Advanced |
+|-----------------------|-----------------------|------------|----------------------------------------------|----------|
+| `illuminance`         | Number:Illuminance    | R          | Illuminance in Lux                           |          |
+
+## Water Sensor
+
+Sensor to detect water leaks.
+
+| Channel               | Type                  | Read/Write | Description                                  | Advanced |
+|-----------------------|-----------------------|------------|----------------------------------------------|----------|
+| `leak`                | Switch                | R          | Water leak detected                          |          |
+| `battery-level`       | Number:Dimensionless  | R          | Battery charge level in percent              |          |
+
+## Contact Sensor
+
+Sensor tracking if windows or doors are open
+
+| Channel               | Type                  | Read/Write | Description                                  | Advanced |
+|-----------------------|-----------------------|------------|----------------------------------------------|----------|
+| `contact`             | Contact               | R          | State if door or window is open or closed    |          |
+| `battery-level`       | Number:Dimensionless  | R          | Battery charge level in percent              |          |
+
+## Air Quality Sensor
+
+Air measure for temperature, humidity and particles.
+
+| Channel               | Type                  | Read/Write | Description                                  | Advanced |
+|-----------------------|-----------------------|------------|----------------------------------------------|----------|
+| `temperature`         | Number:Temperature    | R          | Air Temperature                              |          |
+| `humidity`            | Number:Dimensionless  | R          | Air Humidity                                 |          |
+| `particulate-matter`  | Number:Density        | R          | Category 2.5 particulate matter              |          |
```

Comment:
If this is a particle per million (ppm) channel it should be dimensionless with the unit set to ppm.

Reply:
Really? I still think it's `Density` and with the right [unitHint](https://github.com/openhab/openhab-addons/pull/17719/files#diff-bd10fdfb32ffe023806246406e278b520aed1a4891d95cbe3ec40150748971baR214) in xml it shows correctly ppm in my system.

Reply:
Yes, for reference: https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units

Reply:
Double checked: It's not ppm, it's [μg/m3](https://en.wikipedia.org/wiki/Particulates#European_Union) as described in `channel-types.xml `